### DISCUSSION TITLE
Piloter le chauffage des radiateur thermor et fil pilote heatzy avec versatile thermostat

### DISCUSSION THREAD

**Discussion by Jovaskatv** [jmcollin78/versatile_thermostat#259] [Q&A]
Bonjour a tous ,

je suis un utilisateur home assistant, je ne suis pas un pro, mais j aime la domotique .
je me permet de venir en aide car je n arrive pas a mettre en place tout la configuration .
infos radiateurs: 
pour le salon et salle a manger
thermor 
Ref 4510 51
R4GB-T2SIR   
1500 W

chambre enfant
atlantic 
516015
RWGH EP9
1500W

chambre parents
atlantic 
516020
RWGH EP9
1500W

salle de bain 
Atlantic
850110
RSS02 Electrique
1000W

réglage des températures,  eco 16.5, confort 18.5, boost 20 .
sonde de température humidité type TS0201 en zigbee, pour l extérieur j'utilise la température météo
j ai suivis le tuto cependant je me pose pas certaines questions 
pourquoi bascule le radiateur en mode eco quand la cible est atteinte, le radiateur ce coup jamais dans ce cas la ?
le thermostat passe en sécurité parfois pourquoi ?
pouvez vous m'aider ?

**Comment by jmcollin78** [comment]
Hello @Jovaskatv,

> pourquoi bascule le radiateur en mode eco quand la cible est atteinte

Le Versatile Thermostat (VTherm) ne fait pas ça. Il ne bascule pas en preset Eco quand la cible est atteinte. C'est forcément quelque-chose d'autre fait ça (une automatisation ? le Scheduler ?).

Pour le mode sécurité, tu vas avoir plein d'informations ici : https://github.com/jmcollin78/versatile_thermostat/discussions/238#discussioncomment-7806856

**Reply by jmcollin78** [reply]
Plus d'informations ici : https://github.com/jmcollin78/versatile_thermostat/blob/main/README-fr.md#pourquoi-mon-versatile-thermostat-se-met-en-securite-

**Reply by Jovaskatv** [reply]
Bonjour @jmcollin78 ,

Merci pour le retour. j ai créer des switchs pour chaque radiateurs comme dans la solution ci dessous de @gael
L'utilisation d'un Heatzy est possible à la condition d'utiliser un switch virtuel sur ce modèle :
````
-  platform: template
   switches:
     parents_heating:
       unique_id: heating_parents
       friendly_name: parents heating
       value_template: "{{ is_state_attr('climate.chambre_parents', 'preset_mode', 'comfort') }}"
       icon_template: >-
         {% if is_state_attr('climate.chambre_parents', 'preset_mode', 'comfort') %}
           mdi:radiator
         {% elif is_state_attr('climate.chambre_parents', 'preset_mode', 'away') %}
           mdi:snowflake
         {% else %}
           mdi:radiator-disabled
         {% endif %}
       turn_on:
         service: climate.set_preset_mode
         entity_id: climate.chambre_parents
         data:
           preset_mode: "comfort"
       turn_off:
         service: climate.set_preset_mode
         entity_id: climate.chambre_parents
         data:
           preset_mode: "eco"
````

cependant en observant l'application heatzy et le radiateur,  quand la température cible atteinte la bascule en mode eco fonctionne bien , mais ce mode eco ne coupe pas mon radiateur mais seulement la consigne de chauffe a -2° c  de la température confort .
mon explication n est peux être pas très claire .

**Reply by jmcollin78** [reply]
Tu devrais utiliser les balises de mises en forme du code <> dans l'éditeur. C'est illisible sous cette forme.

**Reply by Jovaskatv** [reply]
Oui désolé, j 'ai apporter la modification .

**Reply by gael1980** [reply]
Bonjour,
A la place de "eco" tu mets "away", cela ne coupe pas le chauffage, mais le mets en hors gel.

```
       turn_off:
         service: climate.set_preset_mode
         entity_id: climate.chambre_parents
         data:
           preset_mode: "away"
```

Si tu souhaites vraiment le couper (déconseiller) 
tu remplaces par : 

```
       turn_off:
         service: climate.turn_off
         entity_id: climate.chambre_parents
```

Gaël

**Comment by jmcollin78** [comment]
> cependant en observant l'application heatzy et le radiateur, quand la température cible atteinte la bascule en mode eco fonctionne bien , mais ce mode eco ne coupe pas mon radiateur mais seulement la consigne de chauffe a -2° c de la température confort .
> mon explication n est peux être pas très claire .

Je n'ai pas compris désolé. Est-ce que tu peux formuler de la façon suivante:

1. Je constate que ....

2. J'aurai voulu que ...
